src/main/java/nl/tudelft/ti2206/group9/audio/SoundEffectPlayer.java

```diff
@@ -106,4 +112,19 @@ public final void setPath(final String location) {
         }
     }
 
+    @Override
+    public void setVolume(final double volumeLevel) {
+        try {
+        audioClip.setVolume(volumeLevel);
+        } catch (NullPointerException ne) { // NOPMD
+            // This try-catch block is just here for testing.
+            // The setVolume method can result in a NullPointer (according to
+            // JUnit), because JUnit can't really play audio neither can Travis.
```

Comment:
Okay, I can live with that sort of catching :) I agree with your argument.
Will change.

<review,logic